runtime/interpreter/interpreter.go

```diff
@@ -3278,13 +3278,6 @@ func (interpreter *Interpreter) newStorageIterationFunction(
 			for key, value := storageIterator.Next(); key != "" && value != nil; key, value = storageIterator.Next() {
 				staticType := value.StaticType(inter)
 
-				// Perform a forced type loading to see if the underlying type is not broken.
-				// If broken, skip this value from the iteration.
-				typeError := inter.checkTypeLoading(staticType)
```

Comment:
Can also remove `checkTypeLoading`

Reply:
Removed in 7229d4451090972d0b6aa24c5a7e2cf0b1637b14